sdk/keyvault/keyvault-keys/review/keyvault-keys.review.api.md

```diff
@@ -0,0 +1,295 @@
+    hsm?: boolean;
+    keySize?: number;
+}
+
+// @public
+export class CryptographyClient {
+    constructor(url: string, key: string | JsonWebKey, // keyUrl or JsonWebKey
+    credential: TokenCredential, pipelineOrOptions?: ServiceClientOptions | NewPipelineOptions);
+    protected readonly credential: ServiceClientCredentials | TokenCredential;
+    decrypt(algorithm: JsonWebKeyEncryptionAlgorithm, ciphertext: Uint8Array, options?: DecryptOptions): Promise<DecryptResult>;
+    encrypt(algorithm: JsonWebKeyEncryptionAlgorithm, plaintext: Uint8Array, options?: EncryptOptions): Promise<EncryptResult>;
+    static getDefaultPipeline(credential: ServiceClientCredentials | TokenCredential, pipelineOptions?: NewPipelineOptions): ServiceClientOptions;
+    getKey(options?: GetKeyOptions): Promise<JsonWebKey>;
+    key: string | JsonWebKey;
+    readonly pipeline: ServiceClientOptions;
+    // Warning: (ae-forgotten-export) The symbol "KeySignatureAlgorithm" needs to be exported by the entry point index.d.ts
+    sign(algorithm: KeySignatureAlgorithm, digest: Uint8Array, options?: RequestOptions): Promise<SignResult>;
+    signData(algorithm: KeySignatureAlgorithm, data: Uint8Array, options?: RequestOptions): Promise<SignResult>;
+    unwrapKey(algorithm: KeyWrapAlgorithm, encryptedKey: Uint8Array, options?: RequestOptions): Promise<UnwrapResult>;
+    readonly vaultBaseUrl: string;
+    verify(algorithm: KeySignatureAlgorithm, digest: Uint8Array, signature: Uint8Array, options?: RequestOptions): Promise<VerifyResult>;
+    verifyData(algorithm: KeySignatureAlgorithm, data: Uint8Array, signature: Uint8Array, options?: RequestOptions): Promise<VerifyResult>;
+    wrapKey(algorithm: KeyWrapAlgorithm, key: Uint8Array, options?: RequestOptions): Promise<WrapResult>;
+}
+
+// @public
+export interface DecryptOptions extends RequestOptions {
+    authenticationData?: Uint8Array;
+    authenticationTag?: Uint8Array;
+    iv?: Uint8Array;
+}
+
+// @public
+export interface DecryptResult {
+    algorithm: JsonWebKeyEncryptionAlgorithm;
+    keyID?: string;
+    result: Uint8Array;
+}
+
+// @public
+export interface DeletedKey extends Key {
+    readonly deletedDate?: Date;
+    readonly recoveryId?: string;
+    readonly scheduledPurgeDate?: Date;
+}
+
+// @public
+export type DeletionRecoveryLevel = "Purgeable" | "Recoverable+Purgeable" | "Recoverable" | "Recoverable+ProtectedSubscription";
+
+// @public
+export interface EncryptOptions extends RequestOptions {
+    authenticationData?: Uint8Array;
+    iv?: Uint8Array;
+}
+
+// @public
+export interface EncryptResult {
+    algorithm: JsonWebKeyEncryptionAlgorithm;
+    authenticationData?: Uint8Array;
+    authenticationTag?: string;
+    iv?: Uint8Array;
+    keyID?: string;
+    result: Uint8Array;
+}
+
+// @public
+export interface GetKeyOptions {
+    requestOptions?: msRest.RequestOptionsBase;
+    version?: string;
+}
+
+// @public
+export interface GetKeysOptions {
```

Comment:
isn't this the same as `RequestOptions`? Other methods don't have a type for their options that is equivalent to `RequestOptions`, they just use it instead.